### PR TITLE
Adding read status for posts

### PR DESCRIPTION
**Pull request by ohboyotero**
To be useful, our boards system needs to keep track of which users have read which posts. This PR introduces a many-to-many relationship where presence of a relationship between character and post indicates that the character has read the post (and vice versa, absence indicates they have not read the post).

### CODE REVIEW

File: world/boards/templatetags/boards_tags.py [hunk 1]
```diff
@@ -14,3 +14,7 @@ def render_post_summary(post):
 @register.simple_tag
 def get_board_detail_url(post):
     return reverse('boards:detail', kwargs={'post_id': post.id})
+
+@register.simple_tag
```
Comment (dancerdevin) [inline world/boards/templatetags/boards_tags.py]: Lines 18 to 20 seem to repeat lines 14 to 16. Is there supposed to be a get_board_detail_url and a get_post_detail_url?

Comment (ohboyotero) [reply world/boards/templatetags/boards_tags.py]: Uhhhh, past @ohboyotero was...out of line. Let's just pretend this didn't happen and I'll delete it.

**Comment by ohboyotero** [comment]
FYI @dancerdevin and @elee-p3 I've finally dusted this PR off after...8 months... 😭 and updated it to ensure that it's working and compatible with Evennia 3.x.

Ready for review!

**Review by dancerdevin — APPROVED** [review]
👍 👍 👍 🥳 🎈 🙀

**Pull request merged**